Assets/Scripts/SS3D/Systems/Inventory/Items/Item.cs

```diff
@@ -185,26 +182,21 @@ public void Unfreeze()
                 itemCollider.enabled = true;
             }
         }
-
-        /// <summary>
-        /// Sets if this item is visible or not
-        /// </summary>
+        
+        
         /// <param name="visible">Should the item be visible</param>
         [Server]
         public void SetVisibility(bool visible)
         {
             // TODO: Make this handle multiple renderers, with different states
-            Renderer renderer = GetComponentInChildren<Renderer>();
+            Renderer[] renderers = GetComponentsInChildren<Renderer>();
 
-            if (renderer != null)
+            foreach (Renderer renderer1 in renderers)
```

Comment:
renderer1 is a weird name

Reply:
is childRenderer better? Just renderer is already taken